### ISSUE TITLE
Spinning wheel characters are included in a redirected stdout

### ISSUE THREAD

**Issue by andy-maier** [click-contrib/click-spinner#8]
It is desirable to be able to redirect command output. If a spinner is active, the spinning wheel characters are redirected as well. In case of machine readable output, this is obviously undesired.

Ideally, the spinning wheel would be suppressed if stdout is redirected.

**Comment by andy-maier** [comment]
This issue can be circumvented by the user of `click-spinner` for example as follows:

```
spinner = click_spinner.spinner() if sys.stdout.isatty() else None
try:
    if spinner:
        spinner.start()
    my_cmd()  # your code while spinning
finally:
    if spinner:
        spinner.stop()
```

Nevertheless, it would be good to put that ability into the Spinner class somehow.

**Comment by andy-maier** [comment]
Fixed in PR #9.
